### ISSUE TITLE
Perm banned again

### ISSUE THREAD

**Issue by NegroSorete** [finndev/PokeBuddy#1071]
I get banned without using the bot for 4 hours, i do not use the pokesniper and nothing , so yeah gg

**Comment by Loppone** [comment]
Can you login from your app?

**Comment by NegroSorete** [comment]
"failed to get game data from the server"

**Comment by Demolition1986** [comment]
@NegroSorete or maybe you got a lot reports from normals players for innapropiated actions, and then u got investigated and terminated account.

**Comment by NegroSorete** [comment]
Wtf are talking about? lul

**Comment by Demolition1986** [comment]
@NegroSorete People can report players in game u dont know about it? OMG.

**Comment by Loppone** [comment]
The same error... ouch!

**Comment by NegroSorete** [comment]
@Demolition1986 But the people cant see you in game, unless you have gyms. Lul

**Comment by Demolition1986** [comment]
@NegroSorete I just asking you, maybe you was a crazy guy of farming gym too, or maybe u got from bots really. U from Argentina?

**Comment by NegroSorete** [comment]
I dont care the gyms, yes and you?

**Comment by Demolition1986** [comment]
@NegroSorete Yes, I from Argentina also :D, btw I see a lot of players +30 or more on gyms at Buenos Aires.

**Comment by NegroSorete** [comment]
@Demolition1986  Y para q me hablas en ingles salame, te decia q me re cabio ban guacho

**Comment by elpibeninja** [comment]
che a nadie le pasa que le anda el bot pero no lea agarra ninguna pokeparada?, en necro me hacen soft cada 1 pokeparada y eso que lo tengo con las coordenadas de mi casa

**Comment by NegroSorete** [comment]
@elpibeninja puede q estes baneado de ip. Si estas baneado de ip no te deja agarrar pokeparadas pero si agarrar pokemones. Cuando estes dudando, entra a la aplicacion y fijate si te sigue tomando las pokeparadas.

**Comment by Koitah** [comment]
I have that in one account lvl 20 and i send them an email =DD ahah , and them say to me this because that " error " 


Me: Hello,

Today i try log in to my account and appear a message " cant load data "  and i see i got a invite of your team to play " ingress " . 

What happen to the account for appear that error ? 

Can help me please ??

Thanks 

Best Regards
Niantic

Trainer,

Thanks for contacting us regarding your sign in issue with Pokémon GO. If you’ve signed into Pokémon GO and it appears that gameplay progress has been reset, be assured that your gameplay progress has NOT been lost. It remains safe and intact. You may have inadvertently created a new Pokémon GO account by unintentionally signing in with a different email address or login method.

If you haven’t already, please try the following:
* iOS users: Please review the login tips outlined in the Pokémon GO Help Center (https://support.pokemongo.nianticlabs.com/hc/en-us/articles/224047008).
* Google Account users: If you’ve signed into Pokémon GO using your Google Account and it appears that you’re level 1, you may not be signing into Pokémon GO using the correct Google Account. Please try signing into any alternate accounts you may have used to create your Pokémon GO account.
* All Trainers: Keep in mind, there are two login methods for Pokémon GO. Trainers can use their Google Account or their Pokémon Trainer Club account to create a Pokémon GO account. If you create an account with one login method and later attempt to use the other login method, a second account is created. To resolve this issue, log out of your account then log back in using your original login method.

Please note: for security reasons, we’re unable to provide you with the email address you use to sign into your account.

-NianticOps

**Comment by Koitah** [comment]
Maybe because them see the email of that account ( lvl 20 )  was seconday ( secondary to recover the password maybe, and i forgot to delet them for obtain data and other shi.t lol )  and the other i created is the main account ... 

i dont know :)

**Comment by Koitah** [comment]
My others 2 it's fine.

**Comment by Althirion** [comment]
Early I got the "failed to get data from the server" I al lvl 27, I dont know what to do now...

**Comment by psblackz** [comment]
some have  "failed to get data from the server" and after can login again ?

**Comment by NegroSorete** [comment]
@psblackz yes, but in very rare cases

**Comment by Demolition1986** [comment]
@NegroSorete El error ese de que podes hacer todo menos lo de las pokeparadas que te dice intente luego nuevamente, es el tipico softbban que antes lo sacabamos dando spin de 40 veces en la pokeparada, ahora parece que lo hicieron mas fuerte jaja. O sea que tarde un poco más pero siempre se sale,.

**Comment by Koitah** [comment]
![sem titulo](https://cloud.githubusercontent.com/assets/20827081/17647145/8f69d0a0-61dd-11e6-81ea-d21b2235f288.jpg)

**Comment by kenwang911** [comment]
@Althirion  can you log back in yet?

**Comment by kenwang911** [comment]
@NegroSorete  have you can log into the accouont now?

**Comment by majdak79** [comment]
you put your pokemon into a gym you got reported, the game is over.

**Comment by Koitah** [comment]
That its other account. I go try login in the another have that error " cant load data " in 2 days to see if change something..
